lib/options-manager.js

```diff
@@ -13,7 +12,6 @@ import {cosmiconfig, defaultLoaders} from 'cosmiconfig';
 import micromatch from 'micromatch';
 import JSON5 from 'json5';
 import toAbsoluteGlob from 'to-absolute-glob';
-import stringify from 'json-stable-stringify-without-jsonify';
```

Comment:
There's a reason this one is used.

Reply:
I see, just dropped the commit to remain this dependency.